device/common/include/traccc/seeding/device/impl/select_seeds.ipp

```diff
@@ -153,13 +153,13 @@ inline void select_seeds(
                 scalar seed1_sum = 0;
                 scalar seed2_sum = 0;
 
-                const internal_spacepoint<spacepoint>& ispB1 =
```

Comment:
Why is this being copied?

Reply:
This avoids a (possibly false-positive) compiler warning.